code/modules/power/singularity/singularity.dm

```diff
@@ -335,10 +335,10 @@
 	var/dir2 = 0
 	var/dir3 = 0
 	switch(direction)
-		if(NORTH||SOUTH)
+		if(NORTH, SOUTH)
```

Comment:
These issues might actually be why the singulo sometimes goes through containment...